src/classBased/components/InputTodo.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-alert */
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/state-in-constructor */
```

Comment:
- Please try to limit the use of linters disabling comments, respecting linters' directives improve your code quality in general.

Reply:
OK, Changes completed